scripts/gitlab/gitlab-ci.yml

```diff
@@ -0,0 +1,31 @@
+stages:
+  - build
+  - test
+  - deploy
+
+variables:
+  APP_NAME: Project
+
+before_script:
+  - echo "hello world"
+
+build:
+  stage: build
+  script:
+    - composer install
+    - blt setup:build
```

Comment:
Well in my build step I'm actually composing a docker image, which contains just the static site build, without actually installing it. `blt setup` would also require mysql, and strictly speaking it would not belong to the build stage. (having a site install in there is a valid point though)